lib/rules/lint-no-attr-in-components.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const Rule = require('./base');
+
+module.exports = class LinkNoAttrComponent extends Rule {
+  isComponentTemplate() {
+    let moduleName = this.templateEnvironmentData.moduleName;
+    let regex = RegExp('templates/components/|components/.*/template.hbs');
```

Comment:
Lets store this regexp in module scope (no need to recreate it for each check).

Reply:
We should consider [module unification](https://github.com/emberjs/rfcs/blob/master/text/0143-module-unification.md) here as well. 

I think the following would work `new RegExp('/templates/components|components/.*/template|ui/components|-components/')`, what do you think?

Reply:
I don't really have a lot of experience working with module unification. I could take a look. If you feel that the above regex would work, I would gladly change it.

Reply:
Ya, I think it’s at least pretty close and we can always tweak it if we need to later.